apis/database/v1alpha2/rdsinstance_types.go

```diff
@@ -38,19 +38,19 @@ const (
 	PostgresqlEngine = "postgres"
 )
 
-// SecurityGroupIDReferencerForRDSInstance is an attribute referencer that resolves SecurityGroupID from a referenced SecurityGroup
-type SecurityGroupIDReferencerForRDSInstance struct {
+// VPCSecurityGroupIDReferencerForRDSInstance is an attribute referencer that resolves SecurityGroupID from a referenced SecurityGroup
+type VPCSecurityGroupIDReferencerForRDSInstance struct {
```

Comment:
I understand that this renaming is to make it clear that the referenced security group is for VPC, but it breaks the pattern in all the referencers where follow underlyingReferencer+"ForType" naming pattern.

Reply:
So, there are two fields in RDS's spec now. One is `VPCSecurityGroupIDs` and the other one is `DBSecurityGroups` and they point to [different type of security groups.](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.RDSSecurityGroups.html)

So, it's actually maps to distinct functionality rather than only naming. Though, I wasn't aware of `underlyingReferencer+"ForType"` pattern. How do you think we should proceed here?